platform-hub-api/app/controllers/kubernetes/changeset_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Kubernetes::ChangesetController < ApiJsonController
+
+  before_action :load_changeset
+
+  # GET /kubernetes/tokens-changeset/:cluster
```

Comment:
Initially I used more verbose controller / route names but decided to keep it shorter. Happy to change if you think it makes things much clearer?